app/components/navbar/navbar.component.js

```diff
@@ -4,21 +4,29 @@ import './navbar.scss';
 const navbarComponent = {
   bindings: {},
   template: `
-		<md-toolbar class="navbar">
-			<div class="md-toolbar-tools">
-				<div flex layout="row" layout-align="start center">
-					<img class="ng-hotrod-logo" ng-src="{{ $ctrl.ngHotrodLogo }}" alt="GitHub">
-					<h2 class="active-page-name" md-truncate>{{ $ctrl.NavigationService.currentPageTitle }}</h2>
-				</div>
-				<a 
-					class="github-link" href="https://github.com/xlts-dev/angularjs-material-vite" 
-					layout="row" 
-					layout-align="center center">
-					<img ng-src="{{ $ctrl.githubLogo }}" alt="GitHub">
-					<div>GitHub</div>
-				</a>
-			</div>
-		</md-toolbar>
+	<md-toolbar class="navbar">
+	  <div class="md-toolbar-tools">
+		<div flex layout="row" layout-align="start center">
+	      <img
+            data-testid="ng-hotrod-logo"
+            class="ng-hotrod-logo"
+            ng-src="{{ $ctrl.ngHotrodLogo }}"
+            alt=""
```

Comment:
We should have alt text here that describes this logo.

Reply:
FWIW, we had discussed this with Jarek and concluded that (since this is a purely decorative image) it should have an empty alt text.